### ISSUE TITLE
Account Splitting

### ISSUE THREAD

**Issue by BlackHaloRose** [LegionDark/Issues#249]
Move: Character: Medina 
From: Account: BlackHaloRose
To: Account: Midi

**Comment by TeoTwawki** [comment]
> To: Account: Medina

No account named Medina was found in the database. The account must exist (but have no characters) to move the character to it.

**Comment by BlackHaloRose** [comment]
That's strange, I certainly made the account. Let me try again.

**Comment by TeoTwawki** [comment]
@BlackHaloRose this was almost never seen. Github does not notify anyone of edits. Have to make new posts for anyone to know an issue update happened.
